addons/controllers/clusterbootstrap_controller_test.go

```diff
@@ -158,6 +160,59 @@ var _ = Describe("ClusterBootstrap Reconciler", func() {
 					return false
 				}, waitTimeout, pollingInterval).Should(BeTrue())
 
+				By("cluster should be marked with finalizer")
+				Eventually(func() bool {
+					err := k8sClient.Get(ctx, client.ObjectKeyFromObject(cluster), cluster)
+					if err != nil {
+						return false
+					}
+					return controllerutil.ContainsFinalizer(cluster, addontypes.AddonFinalizer)
+				}, waitTimeout, pollingInterval).Should(BeTrue())
+
+				By("clusterbootstrap should be marked with finalizer")
+				Eventually(func() bool {
+					err := k8sClient.Get(ctx, client.ObjectKeyFromObject(cluster), clusterBootstrap)
+					if err != nil {
+						return false
+					}
+					return controllerutil.ContainsFinalizer(clusterBootstrap, addontypes.AddonFinalizer)
+				}, waitTimeout, pollingInterval).Should(BeTrue())
+
+				By("cluster kubeconfig secret should be marked with finalizer")
+				clusterKubeConfigSecret := &corev1.Secret{}
+				key := client.ObjectKey{Namespace: cluster.Namespace, Name: secret.Name(clusterName, secret.Kubeconfig)}
+				Eventually(func() bool {
+					err := k8sClient.Get(ctx, key, clusterKubeConfigSecret)
+					if err != nil {
+						return false
+					}
+					return controllerutil.ContainsFinalizer(clusterKubeConfigSecret, addontypes.AddonFinalizer)
+				}, waitTimeout, pollingInterval).Should(BeTrue())
+
+				By("installpackage should have been created for each additional package in the clusterBoostrap")
```

Comment:
nitpick: installpackage -> packageinstall, same comment for line 196

Reply:
ack

Reply:
done
fixed in both places